samples/oauth_rails_vethealth/app/views/oauth_callbacks/show_unverified_state.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1 class="display-2">Callback <em>Failed</em></h1>
+<p class="lead">
+    The time of login was sent in the "state" parameter, but the returned state,
```

Comment:
Found an interesting edge case that gets this to show. When I start the flow by clicking "Oauth Login", I go to ID.me. Invariably, I have forgotten to copy the test user's email, so I open the sample app back up in another tab. This seems to reset the state to a new time. When I then go back to the first tab and continue with the flow, I eventually get to the attached view.

Two questions:
1. Is there a way to separate the state that gets sent to account for another tab being opened?
2. Should we change the text that says "Now with the user identify verified... etc." when there is an error case? It is a little confusing because the error message underneath implies the test user was in fact *not* verified.
![Screen Shot 2019-05-14 at 11 38 09 AM](https://user-images.githubusercontent.com/12074778/57712022-e21aca80-763d-11e9-8d47-1569cab12dd3.png)

Reply:
1 - the state i'm trying to match is embedded into the "login with oauth" link.  I reset that state stored in the session on the login page.  it's a cookie so i'm not sure there's a simple way to split that between tabs. i suppose i could not reset the state it if it was generated within the last couple minutes?

2 - Yeah, I can make that page clearer.  I didn't expect you to find it, honestly!

Reply:
So this screenshot is from an old version of the code.  Pull the latest and the error page here is clearer.  I'm still thinking through keeping the state the same for a few minutes before creatinga . new one.

Reply:
So I fixed 1 by using logout to cleanup session information instead of just resetting it on the login page